docs/pages/zero-trust-access/compliance-frameworks/fedramp.mdx

```diff
@@ -305,3 +305,36 @@ To migrate to a FIPS installation, [set the signature algorithm suite](../../ref
 to `fips-v1`.  Then, any CA with an Ed25519 key must undergo a [CA rotation](../management/operations/ca-rotation.mdx).  The 
 command `tctl status` can be used on v17+ to confirm the signature algorithms in use.  After all CA rotations are 
 complete, proceed with installing the FIPS binary.
+
+## Tradeoffs of using the Teleport FIPS binary
+
+Using Teleport's FIPS binary should not be considered a wholesale security improvement.
+It is designed to meet specific regulatory and compliance requirements (such as
+FedRAMP), not to increase security in general-purpose deployments.
+
+There are several tradeoffs when considering using Teleport FIPS builds in
+cases where it is not strictly required.
+
+- **Restricted Cipher Suite Selection**
+
+  The FIPS build only permits ciphers, key exchange mechanisms, and signature
+  algorithms that are approved under FIPS 140-2 and supported by BoringCrypto.
```

Comment:
I would rephrase this just a bit. 

- Don't call out 140-2 specifically. 
- Maybe mention that since the certification process is so arduous that FIPS often lags behind the state of the art in terms of security.